From ee95071b0270b066c92b1a3259a08f52b04df2dd Mon Sep 17 00:00:00 2001
From: matlabbe <matlabbe@gmail.com>
Date: Sat, 12 Dec 2020 19:56:11 -0500
Subject: [PATCH] Update rules

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 674d4bf..06c8b1f 100755
--- a/debian/rules
+++ b/debian/rules
@@ -50,7 +50,7 @@ override_dh_shlibdeps:
 	# in the install tree and source it.  It will set things like
 	# CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/kinetic/setup.sh" ]; then . "/opt/ros/kinetic/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-kinetic-rtabmap//opt/ros/kinetic/lib/
+	dh_shlibdeps -l$(CURDIR)/debian/ros-kinetic-rtabmap//opt/ros/kinetic/lib/:$(CURDIR)/debian/ros-kinetic-rtabmap//opt/ros/kinetic/lib/x86_64-linux-gnu/:$(CURDIR)/debian/ros-kinetic-rtabmap//opt/ros/kinetic/lib/i386-linux-gnu/:$(CURDIR)/debian/ros-kinetic-rtabmap//opt/ros/kinetic/lib/arm-linux-gnueabihf/
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.17.1

