From f92918a3feb099b03604de5fe61d4d082355e905 Mon Sep 17 00:00:00 2001
From: matlabbe <matlabbe@gmail.com>
Date: Sat, 12 Dec 2020 19:32:50 -0500
Subject: [PATCH] Update rules

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 135d451..f508d71 100755
--- a/debian/rules
+++ b/debian/rules
@@ -50,7 +50,7 @@ override_dh_shlibdeps:
 	# in the install tree and source it.  It will set things like
 	# CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/noetic/setup.sh" ]; then . "/opt/ros/noetic/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/
+	dh_shlibdeps -l$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/:$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/x86_64-linux-gnu/:$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/i386-linux-gnu/:$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/arm-linux-gnueabihf/:$(CURDIR)/debian/ros-noetic-rtabmap//opt/ros/noetic/lib/aarch64-linux-gnu/
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.25.1

