From 0cc560f5c1fcbb5c71cce32cc6a24b03eb29c78e Mon Sep 17 00:00:00 2001
From: matlabbe <matlabbe@gmail.com>
Date: Wed, 23 Mar 2016 16:50:10 -0400
Subject: [PATCH 1/4] Update rules

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index f4fe7d0..d591ba9 100755
--- a/debian/rules
+++ b/debian/rules
@@ -52,7 +52,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/jade/setup.sh" ]; then . "/opt/ros/jade/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-jade-rtabmap//opt/ros/jade/lib/
+	dh_shlibdeps -l$(CURDIR)/debian/ros-jade-rtabmap//opt/ros/jade/lib/:$(CURDIR)/debian/ros-jade-rtabmap//opt/ros/jade/lib/x86_64-linux-gnu/
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
1.9.1

