From 962eee8b19227d02db51a5cb7596a5143928d698 Mon Sep 17 00:00:00 2001
From: matlabbe <matlabbe@gmail.com>
Date: Fri, 4 Aug 2017 12:20:47 -0400
Subject: [PATCH 2/2] Update rules

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 8da7234..8991f69 100755
--- a/debian/rules
+++ b/debian/rules
@@ -48,7 +48,7 @@ override_dh_shlibdeps:
 	# in the install tree that was dropped by catkin, and source it.  It will
 	# set things like CMAKE_PREFIX_PATH, PKG_CONFIG_PATH, and PYTHONPATH.
 	if [ -f "/opt/ros/lunar/setup.sh" ]; then . "/opt/ros/lunar/setup.sh"; fi && \
-	dh_shlibdeps -l$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/x86_64-linux-gnu/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/i386-linux-gnu/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/arm-linux-gnueabihf/
+	dh_shlibdeps -l$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/x86_64-linux-gnu/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/i386-linux-gnu/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/arm-linux-gnueabihf/:$(CURDIR)/debian/ros-lunar-rtabmap//opt/ros/lunar/lib/aarch64-linux-gnu/
 
 override_dh_auto_install:
 	# In case we're installing to a non-standard location, look for a setup.sh
-- 
2.7.4

